content/en/data_jobs/databricks.md

```diff
@@ -71,6 +75,10 @@ Datadog can install and manage a global init script in the Databricks workspace.
 
 {{% tab "Manually install a global init script" %}}
 
+<div class="alert alert-warning">
+This setup does not work on Databricks Clusters in <code>Standard (formerly Shared)</code> access mode because global init scripts cannot be installed on those clusters. If you are using clusters with the <code>Standard (formerly Shared)</code> access mode, you must follow the <a href="?tab=manuallyinstallonaspecificcluster#install-the-datadog-agent">Manually install on a specific cluster</a> instructions for installation on those specific clusters.
```

Comment:
```suggestion
This setup does not work on Databricks Clusters in <code>Standard (formerly Shared)</code> access mode, because global init scripts cannot be installed on those clusters. If you are using clusters with the <code>Standard (formerly Shared)</code> access mode, you must follow the <a href="?tab=manuallyinstallonaspecificcluster#install-the-datadog-agent">Manually install on a specific cluster</a> instructions for installation on those specific clusters.
```